src/seedsigner/models/psbt_parser.py

```diff
@@ -390,3 +415,58 @@ def verify_multisig_output(self, descriptor: Descriptor, change_num: int) -> boo
         is_owner = descriptor.owns(output)
         # print(f"{self.psbt.tx.vout[i].script_pubkey.address()} | {output.value} | {is_owner}")
         return is_owner
+
+
+    def fill_zero_fingerprints(self):
```

Comment:
Maybe this should also be underscore-prefixed since it's only ever called within `parse()` and we don't intend for it to be useful in any other context.